css-transitions-2/Overview.bs

```diff
@@ -167,6 +167,78 @@ Each time a new transition is generated, the current value of the (already
 incremented) <a>current transition generation</a> is stored as the
 transition's <dfn>transition generation</dfn>.
 
+## Defining [=before-change style=]: the ''@starting-style'' rule
+
+Transitions can only start during a [=style change event=] for elements which
+has a defined [=before-change style=] established by the previous
+[=style change event=], which means that newly inserted elements or elements
+that change from not [=being rendered=] to being rendered do not start
+transitions. Still, doing transitions starting from the initial rendering is a
+valid use case.
+
+The <dfn at-rule id="at-ruledef-starting-style">@starting-style</dfn> rule is a
+conditional group rule with the implicit condition that it matches if the
+previous [=style change event=] did not establish a [=before-change style=] for
+the element whose styles are being computed. This is used to establish a
+[=before-change style=] for the current [=style change event=].
+
+If any rules in ''@starting-style'' are applied, the resulting style must be
+used as the [=before-change style=] for the current [=style change event=] and
+the computed styles must be re-computed without ''@starting-style'' rules
+applied.
```

Comment:
I think it would be clearer to replace the part of this sentence after the "and" with a separate sentence saying that `@starting-style` does not affect the after-change style.

Reply:
Modified.